### ISSUE TITLE
Link should be accessible in terminal accessibility help

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#15845]
Testing -171862

It says to read more information at a link in the dialog, but shouldn't keyboard users be able to tab and hit enter on that link to open it? I'm trying to think how a keyboard user would open that link. I also cannot seem to copy it with my mouse
